### ISSUE TITLE
Add alt_text to asset thumbnails on the monograph_catalog page

### ISSUE THREAD

**Issue by sethaj** [mlibrary/heliotrope#1326]
Currently alt-text for asset thumbnails on the monograph_catalog page look like:
```
alt="0g354f20t?file=thumbnail"
```

We do collect alt-text for all file_sets, so we should probably use it. In `app/views/monograph_catalog` there are a few places where we have this:
```
render_thumbnail_tag(document, {}, :counter => document_counter_with_offset(document_counter))
```
Make it this (or something like this):
```
render_thumbnail_tag(document, {alt: fs_presenter.alt_text}, :counter => document_counter_with_offset(document_counter))
```

Does it make sense to use the alt-text from an asset/file_set in this way? Or maybe there should be no alt-text for these? Do these thumbnails convey enough information to need alt-text? I guess they do, even though they're quite small they probably still convey information to sighted users and are not just "decorative".

**Comment by jmcglone** [comment]
@sethaj agreed. These are not decorative images and the thumbnail's alt text would help a user decide if they have interest in a particular asset when browsing the media in this way. 

Because this is a required field, we should always have alt text, right?

Just to nitpick the HTML attribute should be `alt`, which it looks like you have correct in the `render_thumbnail_tag`, but in the HTML snippet was `alt-text`.

**Comment by sethaj** [comment]
Oops, yes, fixed.

**Comment by jmcglone** [comment]
See also in hyrax https://github.com/samvera/hyrax/issues/1248
